[ref #113][member]実験募集フォーム 実施予定期間の一行表示等

radcats opened the pull request:
application_helper.rb form_text_fieldに単位をつけられるように修正
                                   fromとtoを一行で表示するための関数を追加
_form.html.erb 予定人数と時間数のフォームの後ろに単位を追加
                         実施予定日のfrom,toを一行で表示

radcats committed:
[ref #113][member]実験募集フォーム 実施予定期間の一行表示等
application_helper.rb form_text_fieldに単位をつけられるように修正
                                   fromとtoを一行で表示するための関数を追加
_form.html.erb 予定人数と時間数のフォームの後ろに単位を追加
                         実施予定日のfrom,toを一行で表示

okabi commented on app/helpers/application_helper.rb:
```diff
@@ -157,7 +181,7 @@ def form_xxxx_tag(xxxx, name, label, size: 8, value: nil, description: nil, exam
           { class: 'form-control' }
       "<div class='form-group'>".html_safe +
-        label(name,label, class: 'col-sm-2 control-label') +
+        label('damy',label, class: 'col-sm-2 control-label') +
```
割とどうでもいいけど、dummy のスペルミスな気がする。

okabi commented on app/views/experiments/_form.html.erb:
```diff
@@ -17,11 +17,10 @@
     <%= link_to_add_field("実験場所追加", f, :ex_places, {}) %>
-    <%= form_text_field f, :expected_participant_count, '予定人数' %>
-    <%= form_text_field f, :duration, '時間数(単位はh)' %>
+    <%= form_text_field f, :expected_participant_count, '予定人数', unit: ' 人' %>
```
`:unit` の先頭の空白は `form_text_field` メソッド内で追加したい。
つまり、引数で渡すときに空白を先頭につけておく設計は良くないと思います。

okabi reviewed (changes requested):
動作はOK、ちょっとだけ気になる点があるので、それだけ修正お願いします。

radcats committed:
[ref #113][member]実験募集フォーム 実施予定期間の一行表示等

(修正点)
application_helper.rb スペルミス修正
                                   unitの前の空白を関数内で追加するように修正
_form.html.erb unit引数から空白を削除

radcats committed:
[ref #113][member]実験募集フォーム 実施予定期間の一行表示等

(修正点)
Gemfile githubのminitestのバージョン指定

okabi reviewed (approved):
👍
